WooCommerce/WooCommerceTests/Tools/CurrencyFormatterTests.swift

```diff
@@ -47,6 +47,26 @@ class CurrencyFormatterTests: XCTestCase {
         XCTAssertEqual(expectedResult, actualResult)
     }
 
+    /// Verifies that the Arabic string value returns an accurate decimal value
+    ///
+    func testArabicNumeralsReturnDecimal() {
```

Comment:
thanks for adding a test for this ❤️ 

super nit: might be nice to use the new naming convention for test cases:
```suggestion
    func test_arabic_numerals_return_decimal() {
```

would be great to have the Given/When/Then structure in the test case as well!